libretroshare/src/retroshare/rsconfig.h

```diff
@@ -195,33 +228,63 @@ class RsConfigNetStatus
         netDhtNetSize = netDhtRsNetSize = 0;
 	}
 
-	RsPeerId		ownId;
-	std::string		ownName;
+	RsPeerId	ownId;
+	std::string	ownName;
 
-	std::string		localAddr;
+	std::string	localAddr;
 	int			localPort;
-	std::string		extAddr;
+	std::string	extAddr;
 	int			extPort;
-	std::string		extDynDns;
+	std::string	extDynDns;
 
-	bool			firewalled;
-	bool			forwardPort;
+	bool		firewalled;
+	bool		forwardPort;
 
 	/* older data types */
-	bool			DHTActive;
-	bool			uPnPActive;
+	bool		DHTActive;
+	bool		uPnPActive;
 
 	int			uPnPState;
 
 	/* Flags for Network Status */
-	bool 			netLocalOk;     /* That we've talked to someone! */
-	bool			netUpnpOk; /* upnp is enabled and active */
-	bool			netDhtOk;  /* response from dht */
-	bool			netStunOk;  /* recvd stun / udp packets */
-	bool			netExtAddressOk;  /* from Dht/Stun or External IP Finder */
-
-	uint32_t		netDhtNetSize;  /* response from dht */
-	uint32_t		netDhtRsNetSize;  /* response from dht */
+	bool 		netLocalOk;		/* That we've talked to someone! */
+	bool		netUpnpOk;		/* upnp is enabled and active */
+	bool		netDhtOk;		/* response from dht */
+	bool		netStunOk;		/* recvd stun / udp packets */
+	bool		netExtAddressOk;/* from Dht/Stun or External IP Finder */
+
+	uint32_t	netDhtNetSize;	/* response from dht */
+	uint32_t	netDhtRsNetSize;/* response from dht */
+
+	// RsSerializable interface
+public:
```

Comment:
public can be removed here as it is a struct and averything is public by default